bin/MadGraph5_aMCatNLO/gridpack_generation.sh

```diff
@@ -478,20 +478,15 @@ if grep -q -e "\$DEFAULT_PDF_SETS" -e "\$DEFAULT_PDF_MEMBERS" $CARDSDIR/${name}_
         if [ $is5FlavorScheme -eq 1 ]; then
             # 5F PDF
                   sed "s/\$DEFAULT_PDF_SETS/306000,322500,322700,322900,323100,323300,323500,323700,323900,305800,13000,13065,13069,13100,13163,13167,13200,25200,25300,25000,42780,90200,91200,90400,91400,61100,61130,61200,61230,13400,82200,292200,292600,315000,315200,262000,263000/" $CARDSDIR/${name}_run_card.dat > ./Cards/run_card.dat
-            sed -i "s/\$DEFAULT_PDF_MEMBERS/True,  False, False, False, False, False, False, False, False, True,  True,False,False,True, False,False,False,True, True, False,True, True, True, True, True, True, True, True, True, True, True, True,  False, False, False, False, False/" ./Cards/run_card.dat 
+            sed -i "s/\$DEFAULT_PDF_MEMBERS/True,False,False,False,False,False,False,False,False,True,True,False,False,True,False,False,False,True,True,False,True,True,True,True,True,True,True,True,True,True,True,True,False,False,False,False,False/" ./Cards/run_card.dat 
         else
             # 4F PDF
-                  sed "s/\$DEFAULT_PDF_SETS/320900,11082,13091,13191,13202,23100,23300,23490,23600,23790,25410,25510,25570,25605,25620,25710,25770,25805,25840,92000,320900,320500,260400,262400,263400,292000,292400/" $CARDSDIR/${name}_run_card.dat > ./Cards/run_card.dat
-            sed -i "s/\$DEFAULT_PDF_MEMBERS/True,  True, False,False,False,True, True, True, True, True, True, True, True, True, True, True, True, True, True, True, True,  True,  True,  False, False, True,  False/" ./Cards/run_card.dat 
+                  sed "s/\$DEFAULT_PDF_SETS/320600,11082,13091,13191,13202,23100,23300,23490,23600,23790,25410,25510,25570,25605,25620,25710,25770,25805,25840,92000,320900,320500,260400,262400,263400,292000,292400/" $CARDSDIR/${name}_run_card.dat > ./Cards/run_card.dat
+            sed -i "s/\$DEFAULT_PDF_MEMBERS/True,True,False,False,False,True,True,True,True,True,True,True,True,True,True,True,True,True,True,True,True,True,True,False,False,True,False/" ./Cards/run_card.dat 
         fi
     elif [ "$isnlo" -eq "0" ]; then
-        if [ $is5FlavorScheme -eq 1 ]; then
-            # 5F PDF
+        # Same central PDF for both 4 and 5F, variations added at runtime
```

Comment:
I am afraid you misunderstood, read above

Reply:
this is not correct, the central pdf will ebe 4f.
in the original version, for the 4f weights, just replace the first 320900 occurrence with 306000 and keep the second occurrence of 320900, instead of removing the latter

Reply:
Still not clear to me. Can you just list what you want the weights to be or make the change?

Reply:
please revert to the original format

Reply:
I tried again.